packages/server/src/queries/complex/portfolio/__tests__/assets.spec.ts

```
@@ -0,0 +1,305 @@
+    const result = calculatePercentAndFiatValues(
+      MOCK_DATA.categories,
+      assetLists,
+      "total-assets",
+      5
+    ).map((allocation) => ({
+      ...allocation,
+      percentage: allocation.percentage.toString(),
+      fiatValue: allocation.fiatValue.toString(),
+    }));
+
+    delete result[0].asset;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)